easybuild/easyconfigs/e/ECore/ECore-1.5.2.eb

```diff
@@ -12,8 +12,7 @@ sources = [SOURCELOWER_TGZ]
 
 patches = ['ecore-license-var.patch']
 
-comp = ('GCC', '4.6.3')
-dependencies = [('OpenMPI', '1.4.5', '-no-OFED', comp)]
+dependencies = [('OpenMPI', '1.4.5', '-ictce-5.5.0')]
```

Comment:
please renamed ECore easyconfig too, to reflect that this relies on a dependency...